docs/pages/components/product-switch.md

```diff
@@ -28,93 +28,93 @@ Product Switch provides a role based access to all products or LoBs. It shows on
             <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="product-switch-body">
                 <div class="fd-product-switch__body">
                     <ul class="fd-product-switch__list">
-                        <li class="fd-product-switch__item" tabindex="0">
+                        <li class="fd-product-switch__item">
```

Comment:
Did you mean to get rid of `tabindex='0'` on all of these?

Reply:
nope ill reverse them back thanks